release/artifacts.yaml

```diff
@@ -111,8 +111,6 @@
   type: jar-scala
 - target: //ledger/sandbox:sandbox
   type: jar-scala
-- target: //ledger/ledger-api-integration-tests:semantic-test-runner
-  type: jar-deploy
```

Comment:
@bitonic: I'm removing this semantic tests runner from the release. I don't think anybody was using it anyway, and the preferred method to run this is via the ledger-api-tests-tool.